README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 4.21.0
```

Comment:
The code change updates the version number from 4.21.0 to 5.0.0. This change indicates that there may be significant updates or changes in the codebase that warrant a major version increment. It is important to review the release notes or documentation to understand what changes have been made in the new version.